datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -1746,6 +1746,7 @@ mod tests {
     }
 
     #[tokio::test]
+    #[ignore]
```

Comment:
This is failing with a memory exhausted error, I don't believe this is because of an inherent issue with the arrow release, rather a very sensitive test.

Reply:
We should figure out how to update the test to avoid the error, thought -- @kazuyukitanimura  do you have any thoughts on how to do so?

Reply:
We can update the `max_memory` of `new_spill_ctx(2, 1500)` in `check_aggregates` as long as we understand why we need more memory.
Looks like the last change was actually reduced from `2500` to `1500` in #7587

Reply:
I adjusted the memory sizes in 0c4a8a142 to get the tests to pass.

However, I don't have a good idea of what requires more memory now. Any thoughts on allocation changes @tustvold  (like did we change alignment in 50, or add some new fields to the array / buffer structures?)

Reply:
We changed the way aggregates are computed and this might have impacted buffer sizing in some way, it is hard to know for sure without investing a lot of time. If it isn't a major change I wouldn't be overly concerned about it

Reply:
I don't think it is a major change personally